tests/test_time.py

```diff
@@ -5,7 +5,7 @@
 
 class TestTime(unittest.TestCase):
     def test_format_time(self):
-        self.assertEqual(format_time("", {}), "")
+        self.assertEqual(format_time("", {}), None)
```

Comment:
To me these seems like more proper behavior since we would want to treat an empty string like no value exists at all. This works well with other functions in the codebase where we check if arguments are None and if so we remove them from the output.